entity-registry/src/main/java/com/linkedin/metadata/models/registry/config/Event.java

```diff
@@ -0,0 +1,16 @@
+package com.linkedin.metadata.models.registry.config;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.Value;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@Value
+@NoArgsConstructor(force = true, access = AccessLevel.PRIVATE)
```

Comment:
do we need this? thought by having a constructor it removes the default constructor